src/components/Typography/Typography.style.ts

```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components';
+import theme from 'assets/theme';
+import Typography from '@material-ui/core/Typography';
+
+export const Subtitle = styled(Typography).attrs(props => ({
```

Comment:
Per my other comment, "Subtitle" feels like it might be too generic and lead to collisions.